integration-tests/src/test/kotlin/org/stellar/anchor/platform/subtest/PlatformApiTests.kt

```diff
@@ -1,355 +1,4 @@
-package org.stellar.anchor.platform.test
-
-import com.google.gson.reflect.TypeToken
-import org.apache.http.HttpStatus
-import org.junit.jupiter.api.Assertions.assertEquals
-import org.skyscreamer.jsonassert.Customization
-import org.skyscreamer.jsonassert.JSONAssert
-import org.skyscreamer.jsonassert.JSONCompareMode
-import org.skyscreamer.jsonassert.comparator.CustomComparator
-import org.stellar.anchor.api.rpc.RpcRequest
-import org.stellar.anchor.api.rpc.method.NotifyOffchainFundsReceivedRequest
-import org.stellar.anchor.api.rpc.method.RequestOffchainFundsRequest
-import org.stellar.anchor.api.rpc.method.RpcMethod
-import org.stellar.anchor.api.rpc.method.RpcMethod.REQUEST_OFFCHAIN_FUNDS
-import org.stellar.anchor.api.sep.SepTransactionStatus
-import org.stellar.anchor.api.sep.sep12.Sep12PutCustomerRequest
-import org.stellar.anchor.api.sep.sep31.Sep31PostTransactionRequest
-import org.stellar.anchor.apiclient.PlatformApiClient
-import org.stellar.anchor.auth.AuthHelper
-import org.stellar.anchor.platform.Sep12Client
-import org.stellar.anchor.platform.Sep24Client
-import org.stellar.anchor.platform.Sep31Client
-import org.stellar.anchor.platform.TestConfig
-import org.stellar.anchor.util.GsonUtils
-import org.stellar.anchor.util.Sep1Helper.TomlContent
-
-class PlatformApiTests(config: TestConfig, toml: TomlContent, jwt: String) {
```

Comment:
Why are we removing this? Has it been moved to a separate class?

Reply:
Good catch. They were accidentally removed during the migration. Added them back.